contrib/vagrant/provision-util.sh

```diff
@@ -190,7 +190,8 @@ os::provision::get-network-plugin() {
   local default_plugin="${subnet_plugin}"
 
   if [[ "${plugin}" != "${subnet_plugin}" &&
-          "${plugin}" != "${multitenant_plugin}" ]]; then
+          "${plugin}" != "${multitenant_plugin}" &&
+          "${plugin}" != "${cni}" ]]; then
```

Comment:
My intent with defining variables for the plugins was to make it easy to switch the default.  I'm not wedded to validating the plugin, though.  We could as well define a default and blindly attempt to use whatever was specified.  This is dev-only code after all.